ngmix/gmix_nb.py

```diff
@@ -455,14 +455,59 @@ def gmix_fill_cm(gmix, fracdev, TdByTe, Tfactor, pars):
         )
 
 @njit
-def gmix_fill_bdf(gmix, pars):
+def gmix_fill_bd(gmix, pars):
+    """
+    fill a bulge plus disk model
+    """
+
+    row      = pars[0]
+    col      = pars[1]
+    g1       = pars[2]
+    g2       = pars[3]
+    T        = pars[4]
+    lTrat    = pars[5]
+    fracdev  = pars[6]
+    flux     = pars[7]
+
+    TdByTe = 10.0**lTrat
```

Comment:
Log base-10? I feel like this will come back to haunt us...

Reply:
why?

Reply:
Feels unnatural (and is!). I just know I am going to write `np.exp` on this somewhere someday and we will have to explain to people over and over again. But maybe not. Hard to say.

Reply:
I chose it because it is easier to think in base 10 than base e.  If I see logTratio = 1 I know that the T ratio is 10